projects/vdk-plugins/vdk-ingest-http/src/vdk/plugin/ingest_http/ingest_over_http.py

```diff
@@ -102,7 +136,9 @@ def __send_data(self, data, http_url, headers) -> IngestionResult:
                 url=http_url,
                 json=data,
                 headers=headers,
-                verify=False,  # nosec # TODO: disabled temporarily for easier testing, it must be configurable
+                timeout=(self._connect_timeout_seconds, self._read_timeout_seconds),
+                cert=self._cert_file_path,  # certifi.where(),
+                verify=self._verify,
```

Comment:
We don't seem to be guarding against `_verify` being a random string. Is this an acceptable risk?

Reply:
take a look at requests sessions.py verify param pydoc:
```
param verify: (optional) Either a boolean, in which case it controls whether we verify
            the server's TLS certificate, or a string, in which case it must be a path
            to a CA bundle to use. Defaults to ``True``. When set to
            ``False``, requests will accept any TLS certificate presented by
            the server, and will ignore hostname mismatches and/or expired
            certificates, which will make your application vulnerable to
            man-in-the-middle (MitM) attacks. Setting verify to ``False`` 
            may be useful during local development or testing.
```
both boolean or string values are supported

Reply:
My concern was that sessions.request does not really check the value of `verify` if it is a string, https://github.com/psf/requests/blob/main/requests/sessions.py#L707. It just accepts it as-is, which means that a random sting passed as a certificate would cause an exception to be thrown down the chain.
This is a minor thing, though, so you can ignore it.